docs/index.md

```diff
@@ -352,3 +376,22 @@ all internal `.rb` files
 Fast.ruby_files_from(['lib']) # => ["lib/fast.rb"]
 ```
 
+## search_all
+
+Combines the [search_file](#search_file) with [ruby_files_from](#ruby_files_from)
+multiple locations and returns tuples with files and results.
+
+```ruby
+Fast.search_all("(def ast_from_file")
```

Comment:
Missing paren

Reply:
it works in both ways but let me fix to not make it so confusing :D